tsconfig.json

```diff
@@ -0,0 +1,46 @@
+{
+    "compilerOptions": {
+        "target": "es2015",
+        "module": "es2015",
+        "moduleResolution": "node",
+        "jsx": "react",
+        "outDir": "dist",
+        "declaration": false,
+        "rootDir": "src",
+        "allowSyntheticDefaultImports": true,
+        "preserveConstEnums": true,
+        "allowJs": false,
+        "experimentalDecorators": true,
+        "sourceMap": true,
+        "forceConsistentCasingInFileNames": true,
+        "noImplicitReturns": true,
+        "noUnusedParameters": true,
+        "noEmit": false,
+        "skipLibCheck": true,
+        "noImplicitThis": true,
+        "noImplicitAny": true,
+        "strictNullChecks": true,
+        "lib": [
+            "dom",
```

Comment:
I'm not 100% sure about that, but React Native is not a embedded webview like Cordova, so I think the dom lib is not supported. We shouldn't need it in any case.